src/components/add-tag-modal/deps.js

```diff
@@ -19,6 +19,9 @@ export { TAG_HASHTAG, TAG_LOCATION, TAG_SCHOOL, IMPLEMENTED_TAGS } from '../../c
 export { default as ApiClient } from '../../api/client';
 export { API_HOST } from '../../config';
 
+export { Geotags } from '../../prop-types/geotags';
+export { Hashtags } from '../../prop-types/hashtags';
```

Comment:
This should look like `import { Geotags as GeotagsPropTypes } from …`. Otherwise, name is ambigous at this level (and even more so on further levels)